src/shared/os-util.c

```diff
@@ -117,3 +117,26 @@ int load_os_release_pairs(const char *root, char ***ret) {
 
         return load_env_file_pairs(f, p, ret);
 }
+
+int load_os_release_pairs_container_prefix_env(const char *root, char ***ret) {
+        _cleanup_strv_free_ char **os_release_pairs = NULL;
+        char **p, **q, *line;
+        int r;
+
+        r = load_os_release_pairs(root, &os_release_pairs);
+        if (r < 0)
+                return r;
+
+        if (strv_isempty(os_release_pairs))
+                return -1;
```

Comment:
please don't make up errors such as "-1". As per coding style, always return negative errno type error codes.

See: https://systemd.io/CODING_STYLE/#error-handling

That said, why have this check in here anyway? I mean, if the array is empty the loop below will handle things just fine and return an empty array too, no need to exit early with an error ehre.

Reply:
Good point, removed